app/views/sessions/new.html.erb

```diff
@@ -0,0 +1,18 @@
+<% provide(:title, t("log in")) %>
+<h1><%= t "log in" %></h1>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for(:session, url: login_path) do |f| %>
```

Comment:
ngoài form_for còn form nào khác không e

Reply:
Dạ còn form with ạ

Reply:
Phân biệt form+for, form_witth cho chị, trả lời trên box nhé